lib/merlin.ml

```diff
@@ -162,21 +176,11 @@ module Response = struct
         | _ -> failwith "merlin gave bad output")
     | _ -> failwith "merlin gave bad output"
 
-  let crop_timing = function
-    | `Assoc answer -> `Assoc (List.remove_assoc "timing" answer)
-    | _ ->
-        (* Fixme *)
-        failwith
-          "Error while cropping merlin response: reponse should have a key \
-           called timing."
+  let crop_timing answer = crop_arbitrary_keys [ "timing" ] answer
+  let crop_value answer = crop_arbitrary_keys [ "value" ] answer
```

Comment:
Small nitpick/side-note: Given your nice abstraction in `crop_arbitrary_keys` allowing several keys, could you do this in one line?